### ISSUE TITLE
Tab bar background changes colour when using set-colors

### ISSUE THREAD

**Issue by konomikitten** [kovidgoyal/kitty#2510]
Version:
```
$ kitty --version
kitty 0.17.1 created by Kovid Goyal
```
Start kitty:
```
kitty -o allow_remote_control=yes -o enabled_layouts=tall
```
Create a new tab:
```
kitty @ new-window --new-tab
```
Change the colours:
```
kitty @ set-colors -a -c foreground="#d0d0d0" background="#000000" cursor="#d0d0d0" color0="#ffffff" color8="#7f7f7f" color1="#cd0000" color9="#ff0000" color2="#00cd00" color10="#00ff00" color3="#cdcd00" color11="#ffff00" color4="#0000ee" color12="#5c5cff" color5="#cd00cd" color13="#ff00ff" color6="#00cdcd" color14="#00ffff" color7="#e5e5e5" color15="#ffffff"
```
Now the tab bar background is grey instead of black.

**Comment by kovidgoyal** [comment]
As I recall this was fixed in 0.17.2 update. If it still doesn't work
let me know and I will take a look.

**Comment by konomikitten** [comment]
Just confirming that this was fixed when I upgrade to 0.17.3. Thanks for your time and work.
